src/layout_d3util.ts

```diff
@@ -58,7 +58,7 @@ class HTMLLayoutElem extends ALayoutElem implements ILayoutElem {
 
   setBounds(x:number, y:number, w:number, h:number) {
     const unit = this.layoutOption('unit', 'px'),
-      doAnimate = this.layoutOption('animate', false) === true;
+      doAnimate = (this.layoutOption<boolean>('animate', false) === true);
```

Comment:
Do we even need the `=== true`? If the value might have only been true-ish, I could understand, but this seems redundant now?